src/main/java/uk/gov/companieshouse/officer/delta/processor/tranformer/OfficerTransform.java

```diff
@@ -51,16 +51,21 @@ public OfficerAPI transform(OfficersItem source, OfficerAPI officer) throws NonR
             officer.setResignedOn(
                     parseDateString("resignation_date", source.getResignationDate()));
         }
-        officer.setCompanyNumber(source.getCompanyNumber());
-        officer.setTitle(source.getTitle());
-        officer.setForename(source.getForename());
-        officer.setOtherForenames(source.getMiddleName());
-        officer.setSurname(source.getSurname());
-        officer.setHonours(source.getHonours());
 
         final String officerRole = lookupOfficerRole(source.getKind(), source.getCorporateInd());
         officer.setOfficerRole(officerRole);
 
+        if (source.getCorporateInd().equalsIgnoreCase("Y")) {
+            officer.setCompanyNumber(source.getCompanyNumber());
```

Comment:
Do we definitely not want a company number in a natural officer? Just that the perl mapping does have a mandatory company_number field in it:

https://github.com/companieshouse/chs-transform-api/blob/abe90e862ec3e97e0d0a8cbf6a0bbe1d28a45d59/lib/ChsTransformApi/Transforms/Mapping/Officer/Natural.pm

Reply:
I'll change it to match perl, but it does seem pointless seeing as the company number is already stored in the outer object.